lib/iris/plot.py

```diff
@@ -298,6 +298,9 @@ def _draw_2d_from_bounds(draw_method_name, cube, *args, **kwargs):
                     values = np.arange(data.shape[data_dim] + 1) - 0.5
                 else:
                     values = coord.contiguous_bounds()
+                    values = _fixup_dates(coord, values)
+                    if (isinstance(values.ravel()[0], datetime.datetime)):
+                        values = mpl_dates.date2num(values)
```

Comment:
Yes of course it is, sorry, it's green.

In which case, I approve, just need to work out where we are with the image hashing stuff now.